index.bs

```diff
@@ -159,21 +159,19 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
 
 <p>In [[!ISOBMFF]] or [[!HEIF]] files, if an [=AV1 Alpha Image Item=] (respectively. an [=AV1 Alpha Image Sequence=]) is used, the <code>aux_type</code> field of the <code>AuxiliaryTypeProperty</code> (respectively <code>AuxiliaryTypeInfoBox</code>) shall be set to <code>urn:mpeg:mpegB:cicp:systems:auxiliary:alpha</code>, as defined in [[!MIAF]].</p>
 
-  <h2 id="brands-and-file-extensions">Brands and file extensions</h2>
+  <h2 id="brands">Brands, Internet media types and file extensions</h2>
 
 <h3 id="image-and-image-collection-brand">AVIF image and image collection brand</h3>
 Files that conform with the profile-independent restrictions in this document (sections [[#image-item-and-properties]] and [[#alpha-images]]) should include the brand
 <dfn value="" export="" for="AVIF Image brand">avif</dfn> in the [=compatible_brands=] field of the [=FileTypeBox=].
 <p>Files should also carry a compatible brand to identify the AVIF profile (see section [[#profiles]]), if any, with which the file complies.</p>
-<p>If <code>avif</code> is specified in the major_brand field of the FileTypeBox, the file extension should be ".avif".
-The MIME Content Type for files with the ".avif" file extension shall be "image/avif".</p>
+<p>If <code>avif</code> is specified in the major_brand field of the FileTypeBox, the file extension and Internet Media Type should be as defined in [[#avif-mime-definition]].</p>
```

Comment:
The new section says that ".avif", ".heif", or ".hif" is are all allowed valid file extensions, without stating a preference. The nuance that ".avif" is the preferred file extension is lost. Can you fix it (same issue also with the sequence file extension)?